website/versioned_docs/version-1.2/intents/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_label: Intents Overview
+title: Intents Overview
+hide_title: true
+original_id: overview
+---
+
+# Intents Overview
+
+Extending APIs from one application to another is powerful. However, it requires bi-lateral agreements where implementors build to proprietary APIs. A standard language for interaction between applications allows us to create workflows on the fly, so that applications can discover and link to another without any prior knowledge.
+
+
+FDC3 Intents define a standard set of verbs that can be used to put together common cross-application workflows on the financial desktop.
+* Applications register the Intents & [Context Data](../context/overview) combinations they support in the [App Directory](../app-directory/overview)
+* The App Directory supports application discovery by Intents and/or Context Data
+* Intents are not full RPC, apps don’t need to enumerate every function with an intent
+* FDC3 Standard Intents are a limited set, organizations can create their own intents
+
+## Using Intents
+Combined with [Context Data](context/overview) and [App Directory](app-directory/overview) standards, Intents enable rich service discovery on the desktop. For example:
```

Comment:
```suggestion
Combined with [Context Data](../context/overview) and [App Directory](../app-directory/overview) standards, Intents enable rich service discovery on the desktop. For example:
```

The links were wrong, which were contributing to build errors. I have tested to confirm that these new links are correct.

Reply:
fixed